assets/airports/kchs.json

```diff
@@ -0,0 +1,10037 @@
+                "KCHS3": ["PALLI", "SWPFX"],
+                "KCHS15": ["ANGRE", "PALLI", "SWPFX"],
+                "KCHS21": ["ANGRE", "PALLI", "SWPFX"],
+                "KCHS33": ["PALLI", "SWPFX"]
+            },
+            "exitPoints": {
+                "FILLI": ["FILLI"]
+            },
+            "draw": [
+                ["ANGRE", "PALLI", "SWPFX"],
+                ["PALLI", "SWPFX", "FILLI*"]
+            ]
+        }
+    },
+    "stars": {
+        "AMYLU3": {
+            "icao": "AMYLU3",
+            "name": "A my lu Three",
+            "entryPoints": {
+                "RAPZZ": ["RAPZZ"],
+                "MRPIT": ["MRPIT"]
+            },
+            "body": [["AMYLU", "A140+|A230-"], "WILIT", ["CRAAW", "A110+"], "FOURD", "RSRVE"],
+            "rwy": {
+                "KCHS3": [["ROHLO", "A50+|S210"], ["STNNS", "A40|S210"]],
+                "KCHS15": [["KREIS", "A30"]],
+                "KCHS21": [["BAMDE", "A30"]],
+                "KCHS33": [["SNOBB", "A30|S210"]]
+            },
+            "draw": [
+                ["RAPZZ", "AMYLU"],
+                ["MRPIT", "AMYLU"],
+                ["AMYLU", "WILIT", "CRAAW", "FOURD", "RSRVE", "ROHLO", "STNNS"],
+                ["RSRVE", "KREIS"],
+                ["RSRVE", "BAMDE"],
+                ["RSRVE", "SNOBB"]
+            ]
+        },
+        "BAGGY3": {
+            "icao": "BAGGY3",
+            "name": "Baggy Three",
+            "entryPoints": {
+                "SAV": ["SAV", ["DREWE", "A110+"]],
+                "DEQUE": ["DEQUE"]
+            },
+            "body": [["BAGGY", "A70+"],["DDENA", "A50+"]],
+            "rwy": {
+                "KCHS3": [["FEPID", "A30"]],
+                "KCHS15": [["PIPPY", "A50"]],
+                "KCHS21": [["MURAY", "A50"], ["ADERY", "A40"], ["MRISS", "A30"]],
+                "KCHS33": [["CZSAR", "A30"], "#077"]
+            },
+            "draw": [
+                ["SAV", "DREWE", "BAGGY"],
+                ["DEQUE", "BAGGY"],
+                ["BAGGY", "DDENA", "FEPID"],
+                ["DDENA", "PIPPY"],
+                ["DDENA", "MURAY", "ADERY", "MRISS"],
+                ["DDENA", "CZSAR"]
+            ]
+        },
+        "OSPRI7": {
+            "icao": "OSPRI7",
+            "name": "Ospry Seven",
+            "entryPoints": {
+                "ONBEE": ["ONBEE", "NALVE", "COINT", "COBDA", ["NINJA", "A110+|A220-"]],
+                "UNJAM": ["UNJAM", "COBDA",["NINJA", "A110+|A220-"]],
+                "DEFFN": ["DEFFN", "WBODY", "BJETS",["GMAST", "A240+"]],
+                "IRQ":   ["IRQ",["BEEFE", "A240+"], ["BUBOO", "A110+|A220-"]]
+            },
+            "body": [["TRTLS", "A100"], "OSPRI"],
+            "rwy": {
+                "KCHS3" : [["AALEX", "A50"], ["RECHY", "A40|S210"]],
+                "KCHS15": [["LUTHR", "A40"]],
+                "KCHS21": [["LUTHR", "A40"],["MYERS", "A30"]],
+                "KCHS33": [["PAMJE", "A50"],[ "LLEXI", "S250"], ["JAARD", "A30|S210"]]
```

Comment:
All the restrictions here are wrong 😅 
PAMJE should be A50+|A100-|S250
LLEXI should be A50+|S210
JAARD should be A40|S210